addons/account/models/ir_http.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+from odoo import models
+
+
+class IrHttp(models.AbstractModel):
+    _inherit = 'ir.http'
+
+    def session_info(self):
+        result = super(IrHttp, self).session_info()
+        result['is_quick_edit_mode_enabled'] = bool(self.env.company.quick_edit_mode)
```

Comment:
Only add this for internal user
```suggestion
        if self.env.user._is_internal() and self.env.company.quick_edit_mode:
            result['is_quick_edit_mode_enabled'] = True
```